### ISSUE TITLE
Anyone Still Around?

### ISSUE THREAD

**Issue by Skemp45** [RaspberryPints/RaspberryPints#311]
It is been a bit and RaspberryPints has been pretty much running itself on auto pilot. How is everyone doing?

**Comment by mcangeli** [comment]
Been busy. Haven't had time to work any more on the RFID stuff (but its
working...)

**Comment by Skemp45** [comment]
Yeah, I have been getting about 1 support question a week so I know people are still using it haha. I have not decided if I want to undertake a remake or not on this.

**Comment by thadius856** [comment]
I'm out. Going through some pretty major life changes and don't even have
time to brew.

I've made huge leaps on my sql, python, and vba skills through work, but
I'm not at a point right now where I can spare much time.

Let me know if you guys want the GitHub account handed over to someone.

**Comment by Skemp45** [comment]
@thadius856 I will take it over, I still control the web hosting and domain etc. shawn@besmartdesigns.com just email the details on over

**Comment by joahfitzgerald** [comment]
I'm pleased to see this still has some collaboration.   Thanks to all who
have participated thus far.

**Comment by Boombrewer** [comment]
Sorry guys, I'm out.  Too many moves by Uncle Sam and the system does not prove robust against movers.  I'm sad to say I'll be disassembling when I can get some spare time and have a few flow meters and such for sale.  Good luck to you and hope to rejoin when we get a little more stable.

**Comment by Skemp45** [comment]
No worries to everyone who can no longer assist, It has been two years and a lot of changes have come through. What we were able to do as a group of strangers is nothing short of amazing. I plan on taking Rpints and re writing most of the code to current standards and design a whole new layout along with a very very very easy install script. I will most likely start selling pre configured pi's with the new version. I will keep yall updated with any changes along the way.

Best Regards,
Shawn

**Comment by thadius856** [comment]
Just let me know if you do end up selling so I can remove the link from my
HBT account. They said they'd ban me if I was linked to our page and
started selling or collecting donations without a vendor account.

**Comment by RaspberryPints** [comment]
I will for sure, if I go that route I will probably get the proper merchant account too.

**Comment by thadius856** [comment]
@Skemp45 Can you remove me from the help email distro for now?

**Comment by Skemp45** [comment]
Yes of course I will

**Comment by Scimmia22** [comment]
So is this dead now? Issues closed with no resolution, PRs sitting forever...

**Comment by jcorrado** [comment]
Hello,

This project seems to have wound down, but if any of the original devs are around, I just wanted to say thank you as I used your Arduino code in my own, toy project.  Attributed, of course.


Thanks,
Jereme

**Comment by Philusha1983** [comment]
Good to see people are interested.
I'll be happy to see what you made from this code.
I'm following this project for awhile looking for some spare time to
implement it in my kegarator.

**Comment by Skemp45** [comment]
HEY! I am still here just unable to dedicate time to it at the moment. I am very happy you were able to use some of our code! Love to see people making stuff!

**Comment by jcorrado** [comment]
Thanks, @Skemp45  - you guys saved me a ton of time!

@Philusha1983 I just recently began learning Clojure and needed a little project to tackle as an exercise... something that would benefit from a multi-threaded solution.  At around that same time, someone in my office asked me if the coldbrew keg could tweet in the morning, so people knew if they should pick up their own iced coffees or not :)

I build a little HTTP API to surface basic keg stats: [know-the-flow](https://github.com/jcorrado/know-the-flow).
